_docs/setup/kubernetes/helm-install.md

```diff
@@ -74,13 +76,13 @@ Upgrading Istio using Helm is not validated.
     1. With [automatic sidecar injection]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection) (requires Kubernetes >=1.9.0):
 
     ```command
-    $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set sidecar-injector.enabled=true
+    $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set sidecar-injector.enabled=true --set global.proxy.image=proxyv2 --set prometheus.enabled=true
```

Comment:
`prometheus.enabled` is enabled by default in `values.yaml`, do we still need this?

Reply:
good point, i looked at wrong code base.